docs/rhoas/cli-context/README.adoc

```diff
@@ -0,0 +1,240 @@
+//OpenShift Streams for Apache Kafka
+:base-url: https://github.com/redhat-developer/app-services-guides/tree/main/docs/
+:product-long-rhoas: OpenShift Application Services
+:rhoas-cli-base-url: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
+:rhoas-cli-ref-url: commands
+:rhoas-cli-getting-started-url-context: cli-context/
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+:product-kafka: Streams for Apache Kafka
+
+[id="chap-getting-started-rhoas-cli-context"]
+= Getting started with the rhoas CLI for context management
+:context: getting-started-rhoas-context
+
+[role="_abstract"]
+With the {product-long-rhoas} `rhoas` command-line interface (CLI), you can manage service contexts.
+
+As a developer of applications and services,
+you can use `rhoas` to manage service contexts that can be used to generate application configurations.
+Contexts allow you to work with multiple projects using different services easily.
+
+[role="_abstract"]
+A service context is a group of application service instances and their service identifiers.
```

Comment:
I think this should not be context centric (as feature) but more focused on problem we solve. 

Problem we solve:

- Configure,Develop and Deploy your managed services based quarkus apps by using single command


Who we targeting:

- Users familliar with RHOAS CLI
- Users using gitops workflows

Reply:
I will prefer mentioning context enough in this guide, as I believe the command reference won't be enough to make user comfortable with using contexts.

Would like to keep things like creating context and setting services in it.

Reply:
Title of this guide is "Getting started with RHOAS CLI for context management". We should hange it to "Generating configurations with RHOAS CLI" if we would wantcontext management to take a back seat.

Reply:
Definietly +1 on making it more clear on what this guide does.
Generating configurations with RHOAS CLI" sounds like good path forward. Configuring your applications to use managed services etc.

Reply:
> I will prefer mentioning context enough in this guide, as I believe the command reference won't be enough to make user 

I know that we want to promote context but we need to do it to make appealing to end user. We need to talk in guide about problem we solve and target to show outcomes that have any meaning to them.

We have other type of artifact in this repo that are typical docs reference/concept style docs. Guides are about users use cases that red hat want to promote. Mainly cloud native and end to end integration thru software and kuberbetes. I see blog post giving us more freedom. Guides should really cover red hat strategic goals around managed services in clear way. This means that rhoas context guide needs to be presented as Configuring Managed Services using cli guide

Reply:
Same we need to be consistent in vision and rename existing cloud native guide to "Configuring Managed Services Using Service Binding Operator". 

This leaves clear path for all users:

- UI user : I will copy urls and other data myself by navigating in UI
- CLI user: Rhoas context 
-  SBO user: Rhoas cluster